src/site/_includes/components/EventTable.js

```diff
@@ -112,8 +113,24 @@ module.exports = (days, authorsCollection) => {
     `;
   };
 
+  const renderCarouselDay = ({title, videoId}, index) => {
+    const previewImage = videoId ? html`<img src="https://img.youtube.com/vi/${videoId}/maxresdefault.jpg"` : '';
```

Comment:
I can't recall the thumbnail size but i think we can request smaller versions.

default.jpg <- 120 x 90
mqdefault.jpg <- 320 x 180